pkg/issuer/venafi/client/venaficlient.go

```diff
@@ -67,45 +70,38 @@ type connector interface {
 	RenewCertificate(req *certificate.RenewalRequest) (requestID string, err error)
 }
 
-func New(namespace string, secretsLister corelisters.SecretLister,
-	issuer cmapi.GenericIssuer) (Interface, error) {
-
+func New(namespace string, secretsLister corelisters.SecretLister, issuer cmapi.GenericIssuer) (Interface, error) {
 	cfg, err := configForIssuer(issuer, secretsLister, namespace)
 	if err != nil {
 		return nil, err
 	}
 
-	client, err := vcert.NewClient(cfg)
+	vcertClient, err := vcert.NewClient(cfg)
 	if err != nil {
 		return nil, fmt.Errorf("error creating Venafi client: %s", err.Error())
 	}
 
 	return &Venafi{
 		namespace:     namespace,
 		secretsLister: secretsLister,
-		client:        client,
+		vcertClient:   vcertClient,
 	}, nil
 }
 
 // configForIssuer will convert a cert-manager Venafi issuer into a vcert.Config
 // that can be used to instantiate an API client.
 func configForIssuer(iss cmapi.GenericIssuer, secretsLister corelisters.SecretLister, namespace string) (*vcert.Config, error) {
 	venCfg := iss.GetSpec().Venafi
-	switch {
-	case venCfg.TPP != nil:
```

Comment:
Why the switch away from `switch`? It makes it clear when reading that only one of these cases can be met, but maybe I'm missing something :)